### DISCUSSION TITLE
Weird display of array of images

### DISCUSSION THREAD

**Discussion by CarloLucibello** [fonsp/Pluto.jl#2097] [Questions about using Pluto] [answered]
A single  image (a `Matrix{Grey}`) is displayed fine in Pluto (although seeing does grid lines is a bit annoying):

![pluto1](https://user-images.githubusercontent.com/7014210/166109565-8ff2086c-debc-493c-b040-95eff2ced723.png)

But if a put a few of them in an array (or also a tuple) I see some visual artifacts:

![pluto2](https://user-images.githubusercontent.com/7014210/166109584-58b0642b-32a8-40fd-8f90-2e03618b149d.png)

**Comment by fonsp** [comment]
Can you share a notebook file?

**Comment by fonsp** [comment]
You may need to add `using ImageShow` to get a nicer display

**Comment by CarloLucibello** [comment, marked as answer]
`using ImageShow, ImageIO` fixes the problem.

![Screenshot from 2022-05-08 13-10-37](https://user-images.githubusercontent.com/7014210/167293711-67fc6fc5-d5e9-4971-8eb5-4dc30bb2bc3b.png)

[Here](https://gist.github.com/CarloLucibello/f3f1b354cd14323281e2a90513f30118) 2 notebooks (w/ and w/o ImageShow)
